transmogrifier/sources/xml/datacite.py

```diff
@@ -302,20 +246,117 @@ def get_optional_fields(self, xml: Tag) -> dict | None:
         return fields
 
     @classmethod
-    def get_main_titles(cls, xml: Tag) -> list[str]:
+    def get_alternate_titles(
+        cls, source_record: Tag
+    ) -> list[timdex.AlternateTitle] | None:
+        alternate_titles = []
+        alternate_titles.extend(
+            [
+                timdex.AlternateTitle(
+                    value=str(title.string.strip()),
+                    kind=title["titleType"],
+                )
+                for title in source_record.find_all("title", string=True)
+                if title.get("titleType")
+            ]
+        )
+        alternate_titles.extend(list(cls._get_additional_titles(source_record)))
+        return alternate_titles or None
+
+    @classmethod
+    def _get_additional_titles(
+        cls, source_record: Tag
+    ) -> Iterator[timdex.AlternateTitle]:
+        """Get additional titles from get_main_titles method."""
+        for index, title in enumerate(cls.get_main_titles(source_record)):
+            if index > 0:
+                yield timdex.AlternateTitle(value=title)
+
+    @classmethod
+    def get_content_type(cls, source_record: Tag) -> list[str] | None:
+        content_types = []
+        if resource_type := source_record.metadata.find("resourceType"):
+            if content_type := resource_type.get("resourceTypeGeneral"):
+                if cls.valid_content_types([content_type]):
+                    content_types.append(str(content_type))
```

Comment:
Small update but perhaps rewrite to: 

```
content_types.append(str(content_type)
if cls.valid_content_types(content_types):
    return content_types
else: 
    <...>
```
This would remove the use of `or None` but might be a smoother read?